src/main/java/duke/task/TaskList.java

```diff
@@ -0,0 +1,147 @@
+package duke.task;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * TaskList class is used to store different Task objects and
+ * allows the user to modify the list of tasks through
+ * this class.
+ */
+public class TaskList {
+
+    private List<Task> list;
+    private int activeTasks = 0;
+    private int completedTasks = 0;
+
+    public TaskList() {
+        this.list = new ArrayList<>();
+    }
+
+    private TaskList(List<Task> list) {
+        this.list = list;
+        for (Task t : this.list) {
+            if (t.isDone()) {
+                this.completedTasks += 1;
+            } else {
+                this.activeTasks += 1;
+            }
+        }
+    }
+
+    public List<Task> getList() {
+        return this.list;
+    }
+
+    /**
+     * Adds a Task to the list.
+     * @param item Task object to be added on to the list.
+     * @return String result that describes the item added
+     * if successful.
+     */
+    public String addItem(Task item) {
```

Comment:
Should the method be renamed as addTask instead to be consistent?

Reply:
Yes, I totally agree with kerkpy's input